src/app/snapshots/[id]/page.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { Account, Space } from '@storacha/ui-react'
+
+import { withAuthContext, withData } from '@/../.storybook/decorators'
+
+import Page from './SnapshotPage'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  // Uses division slash (∕) instead of regular slash (/) in the title.
+  title: '/snapshots∕[id]',
```

Comment:
```suggestion
  title: '∕snapshots∕[id]',
```

(Wrong slash. Hacky solution is hacky. 😛)